docs/user-guide/CAI.rst

```diff
@@ -0,0 +1,313 @@
+.. include:: /content/nemo.rsts
+
+.. _model-aligner-cai:
+
+Constitutional AI: Harmlessness from AI Feedback
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+
+
+`Constitutional AI (CAI)`_ is an approach by Anthropic to train AI systems that are helpful, honest, and harmless, without relying on human feedback labels for harmlessness.
+CAI uses a set of natural language principles to guide AI behavior and self-improvement.
+The method consists of two stages: a supervised learning stage, where the AI critiques and revises its own responses according to the principles, and a reinforcement learning stage, where the AI learns from its own feedback based on the principles.
+CAI allows training a harmless, but non-evasive AI assistant that engages with harmful queries by explaining its objections to them.
+
+.. _Constitutional AI (CAI): https://arxiv.org/abs/2212.08073
+
+CAI
+###############
+The basic steps of CAI are described in this section and illustrated in the figure below (`Figure 1 <https://arxiv.org/abs/2212.08073>`_).
```

Comment:
The basic steps of CAI are described in this section and illustrated in `Figure 1 <https://arxiv.org/abs/2212.08073>`_).

Reply:
I think I didn't explain myself well enough in the previous discussion, sorry about that. 

By referring to Figure 1, I'm not referring to Figure 1 in this specific document, but am referring to Figure 1 from CAI paper by anthropic. I.e. this is more of a citation, trying to make sure that the user understands that this is not a figure that we've made, but is taken from the paper.

See Figure 1 here. - https://arxiv.org/pdf/2212.08073.
<img width="640" alt="image" src="https://github.com/NVIDIA/NeMo-Aligner/assets/150582838/d07b3d8e-91bc-44d4-8c9e-8a3bd904bed6">

How about the following to make this clearer:
The basic steps of CAI are described in this section and illustrated in the figure below (`Figure 1 <https://arxiv.org/abs/2212.08073>`_, Bai et. al).

Reply:
OK, thank you for the clarification. The text as written is correct then.